chemprop/data/data.py

```diff
@@ -267,6 +324,18 @@ def atom_descriptors(self) -> List[np.ndarray]:
 
         return [d.atom_descriptors for d in self._data]
 
+    def bond_descriptors(self) -> List[np.ndarray]:
+        """
+        Returns the atom descriptors associated with each molecule (if they exit).
```

Comment:
Change description from "atom" to "bond"

Reply:
done!